packages/react/src/components/List/List.story.jsx

```diff
@@ -253,6 +253,27 @@ export const WithSecondaryValue = () => (
 
 WithSecondaryValue.storyName = 'with secondaryValue';
 
+export const WithSecondaryValueAsObject = () => (
```

Comment:
I don't think we need this story since we you also created the new ListItem story